src/integrationTest/java/uk/gov/hmcts/darts/event/controller/EventsControllerTest.java

```diff
@@ -37,6 +39,7 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 import static uk.gov.hmcts.darts.common.enums.SecurityRoleEnum.CPP;
 import static uk.gov.hmcts.darts.common.enums.SecurityRoleEnum.SUPER_ADMIN;
+import static uk.gov.hmcts.darts.test.common.TestUtils.UUID_REGEX;
 
 @AutoConfigureMockMvc
```

Comment:
Add test to ensure we only process for Admins and it fails for others

Reply:
Added